code/game/objects/items/storage/backpack.dm

```diff
@@ -444,8 +466,7 @@
 	name = "\improper USCM technician welderpack"
 	desc = "A specialized backpack worn by USCM technicians. It carries a fueltank for quick welder refueling and use,"
 	icon_state = "engineerpack"
-	var/max_fuel = 260
-	max_storage_space = 15
```

Comment:
This buff is not necessary.

Reply:
Which is why the welderpack is such a popular choice, right?

Reply:
Yea, welderpack is kind of the standard engie gear. I see few engies that know what they are doing without one.

Reply:
They'll take it along as a fuel repository, but not as something they actually wear.

Reply:
It's a satchel-sized backpack with fuel in it. There are trade-offs.